tests/golden_outputs/t0_1_2_vcf_at_0_with_DP_filter

```diff
@@ -0,0 +1,123 @@
+##contig=<ID=GL000231.1,length=27386>
+##contig=<ID=GL000210.1,length=27682>
+##contig=<ID=GL000239.1,length=33824>
+##contig=<ID=GL000235.1,length=34474>
+##contig=<ID=GL000201.1,length=36148>
+##contig=<ID=GL000247.1,length=36422>
+##contig=<ID=GL000245.1,length=36651>
+##contig=<ID=GL000197.1,length=37175>
+##contig=<ID=GL000203.1,length=37498>
+##contig=<ID=GL000246.1,length=38154>
+##contig=<ID=GL000249.1,length=38502>
+##contig=<ID=GL000196.1,length=38914>
+##contig=<ID=GL000248.1,length=39786>
+##contig=<ID=GL000244.1,length=39929>
+##contig=<ID=GL000238.1,length=39939>
+##contig=<ID=GL000202.1,length=40103>
+##contig=<ID=GL000234.1,length=40531>
+##contig=<ID=GL000232.1,length=40652>
+##contig=<ID=GL000206.1,length=41001>
+##contig=<ID=GL000240.1,length=41933>
+##contig=<ID=GL000236.1,length=41934>
+##contig=<ID=GL000241.1,length=42152>
+##contig=<ID=GL000243.1,length=43341>
+##contig=<ID=GL000242.1,length=43523>
+##contig=<ID=GL000230.1,length=43691>
+##contig=<ID=GL000237.1,length=45867>
+##contig=<ID=GL000233.1,length=45941>
+##contig=<ID=GL000204.1,length=81310>
+##contig=<ID=GL000198.1,length=90085>
+##contig=<ID=GL000208.1,length=92689>
+##contig=<ID=GL000191.1,length=106433>
+##contig=<ID=GL000227.1,length=128374>
+##contig=<ID=GL000228.1,length=129120>
+##contig=<ID=GL000214.1,length=137718>
+##contig=<ID=GL000221.1,length=155397>
+##contig=<ID=GL000209.1,length=159169>
+##contig=<ID=GL000218.1,length=161147>
+##contig=<ID=GL000220.1,length=161802>
+##contig=<ID=GL000213.1,length=164239>
+##contig=<ID=GL000211.1,length=166566>
+##contig=<ID=GL000199.1,length=169874>
+##contig=<ID=GL000217.1,length=172149>
+##contig=<ID=GL000216.1,length=172294>
+##contig=<ID=GL000215.1,length=172545>
+##contig=<ID=GL000205.1,length=174588>
+##contig=<ID=GL000219.1,length=179198>
+##contig=<ID=GL000224.1,length=179693>
+##contig=<ID=GL000223.1,length=180455>
+##contig=<ID=GL000195.1,length=182896>
+##contig=<ID=GL000212.1,length=186858>
+##contig=<ID=GL000222.1,length=186861>
+##contig=<ID=GL000200.1,length=187035>
+##contig=<ID=GL000193.1,length=189789>
+##contig=<ID=GL000194.1,length=191469>
+##contig=<ID=GL000225.1,length=211173>
+##contig=<ID=GL000192.1,length=547496>
+##contig=<ID=NC_007605,length=171823>
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	HG00141	HG01958	HG01530
+1	12141	.	C	<NON_REF>	.	.	END=12144	GT:GQ:PL:MIN_DP:DP	./.:0:0,0,0:0:2	.:.:.:.:.	.:.:.:.:.
+1	12145	.	C	<NON_REF>	.	.	END=12277	GT:GQ:PL:MIN_DP:DP	./.:0:0,0,0:0:2	./.:0:0,0,0:0:3	.:.:.:.:.
+1	12278	.	C	<NON_REF>	.	.	END=12295	GT:GQ:PL:MIN_DP:DP	./.:0:0,0,0:0:2	.:.:.:.:.	.:.:.:.:.
+1	17385	.	G	A,T,<NON_REF>	.	.	BaseQRankSum=-2.074;ClippingRankSum=0.555;MQRankSum=-0.329;ReadPosRankSum=0.005;MQ=31.72;RAW_MQ=8.0;MQ0=8;DP=200	GT:GQ:SB:AD:PL:PGT:PID:DP	./.:99:58,0,22,0:58,22,17,17:504,0,9807,678,1870,2548,678,1870,2548,2548:0|1:17385_G_A:80	./.:99:0,0,0,0:0,37,120,37:3336,4536,7349,358,958,0,4536,7349,958,7349:0|1:17385_G_T:120	.:.:.:.:.:.:.:.
```

Comment:
Sorta nit-picky, but maybe we should augment these golden output filenames to make clear what the filter is exactly? Am I understanding correctly that these are golden outputs for DP > 100? Because this file shows results less than 100 and greater than 100...?

Reply:
Yes, these are golden outputs for DP>100. For attributes that have empty values, the expressions evaluate to true currently.
Good idea on getting these outputs annotated, I will get in touch tomorrow for a quick tutorial on how to do it.